psse/psse-converter/src/main/java/com/powsybl/psse/converter/PsseImporter.java

```diff
@@ -65,15 +66,31 @@ public String getComment() {
         return "PSS/E Format to IIDM converter";
     }
 
+    private String findExtension(ReadOnlyDataSource dataSource, boolean throwException) throws IOException {
+        for (String ext : EXTS) {
+            if (dataSource.exists(null, ext)) {
+                return ext;
+            }
+        }
+        if (throwException) {
+            throw new PsseException("File " + dataSource.getBaseName()
+                    + "." + Joiner.on("|").join(EXTS) + " not found");
+        }
+        return null;
+    }
+
     @Override
     public boolean exists(ReadOnlyDataSource dataSource) {
         try {
-            if (dataSource.exists(null, EXT)) {
-                try (BufferedReader reader = new BufferedReader(new InputStreamReader(dataSource.newInputStream(null, EXT)))) {
-                    String titleLine = reader.readLine();
-                    if (titleLine != null) {
-                        return titleLine.length() >= 38
-                                && titleLine.charAt(37) == '/';
+            String ext = findExtension(dataSource, false);
+            if (ext != null) {
+                try (BufferedReader reader = new BufferedReader(new InputStreamReader(dataSource.newInputStream(null, ext)))) {
```

Comment:
Have you tried what you get in the case of a *.raw|RAW file but the content is not a raw file?

Reply:
Ideally we should had unit tests to cover the different cases (real raw or RAW, fake one)

Reply:
raw is a really common extension, used for instance in photography. I think it's a good idea to check if the file is valid or not.

Reply:
I have added a test for a better check of the header

Reply:
I added an additional unit test to cover @mathbagu use case so a "fake" .raw file.